src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php

```diff
@@ -26,25 +28,23 @@ public function __construct()
 
         $this
             ->desc('Get stats for projects')
+            ->param('commit', false, new boolean(true), 'Commit  project deletion', true)
```

Comment:
Remove extra space. Also shouldn't boolean be Boolean?

```suggestion
            ->param('commit', false, new Boolean(true), 'Commit project deletion', true)
```